src/components/DeviceRequest/DeviceRequest.js

```diff
@@ -1,11 +1,207 @@
-import React from "react";
-import { Appbar } from "..";
+import React, { useState, useEffect } from "react";
+import { Appbar, Select, Table } from "..";
 import { adminSubRouters } from "../../config/routes";
+import "../styles/deviceRequest.css";
+import { tableOptions } from "../../config/options/options";
+import { useQuery } from "../../utils/routerHandler";
+import ENV_CONFIG from "../../config";
+import { useHistory, useLocation } from "react-router";
+import { parseSortOption } from "../../utils/parser";
+import { sortByElementProperty } from "../../utils/sort";
+import {
+  getDeviceInfoOfEmployeeByEmail,
+  getDeviceRequests,
+  updateEmployeeDeviceInfo,
+  updateStatusOfDeviceRequest,
+} from "../../apiService";
+import toast from "react-hot-toast";
+import { useDispatch, useSelector } from "react-redux";
+import { deviceRequestAction } from "../../store/actions";
+
+const tableHeaders = [
+  "Name",
+  "Team",
+  "Created Date",
+  "Device",
+  "Notification",
+  "",
+];
 
 function DeviceRequest() {
+  const [reload, setReload] = useState(Math.random());
```

Comment:
What is the purpose of this Math.random?

Reply:
It isn't necessary.